webview-ui/src/components/settings/ThinkingBudget.tsx

```diff
@@ -8,16 +8,10 @@ interface ThinkingBudgetProps {
 	apiConfiguration: ApiConfiguration
 	setApiConfigurationField: <K extends keyof ApiConfiguration>(field: K, value: ApiConfiguration[K]) => void
 	modelInfo?: ModelInfo
-	provider?: ApiProvider
 }
 
-export const ThinkingBudget = ({
-	apiConfiguration,
-	setApiConfigurationField,
-	modelInfo,
-	provider,
-}: ThinkingBudgetProps) => {
-	const tokens = apiConfiguration?.modelMaxTokens || modelInfo?.maxTokens || 64_000
+export const ThinkingBudget = ({ apiConfiguration, setApiConfigurationField, modelInfo }: ThinkingBudgetProps) => {
+	const tokens = apiConfiguration?.modelMaxTokens || 16_384
```

Comment:
This is what changes the default for the slider; we default to 16K instead of `modelInfo.maxTokens`.

Reply:
If the max output tokens is 16k, I think we'd want to default this to lower. 8k maybe?

Reply:
Max output will be 64K since we only show this for the thinking variant.

Reply:
Sorry I’m slow on this - what are the defaults for max tokens and thinking budget then for the thinking variant?

Reply:
When you are presented with the sliders for `maxTokens` and `thinkingBudgetTokens` the defaults will be `16_384` and `Math.floor(0.8 * 16_384)` respectively.